packages/google_sign_in/google_sign_in_android/example/android/app/src/androidTest/java/io/flutter/plugins/googlesigninexample/GoogleSignInTest.java

```diff
@@ -8,10 +8,13 @@
 
 import androidx.test.core.app.ActivityScenario;
 import io.flutter.plugins.googlesignin.GoogleSignInPlugin;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class GoogleSignInTest {
   @Test
+  @Ignore(
```

Comment:
Is this equivalent to disabling the test?

Reply:
Yea it causes this test to be skipped. I verified the test runs locally on the latest version of Flutter, so I think it is a problem with how we run FTL tests.